java/com.domain/src/main/resources/application.yml

```diff
@@ -0,0 +1,29 @@
+server:
+  tomcat:
+    uri-encoding: UTF-8
+  port:  8080
+  servlet:
+    context-path: /
+spring:
+  freemarker:
+    enabled: false
+  thymeleaf:
+    enabled: false
+  application:
+    name: api-domain
+  jackson:
+    default-property-inclusion: NON_NULL
+domain:
+  url: http://ttt.cn/
+  limit: 100.0
+  authentication:
+    enabled: true
+    token: e3853146-a77e-4f51-b52d-277f719a5849
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 4b6a4754-c416-491a-8b4a-603428fbf873 -->